pkg/client/config.go

```diff
@@ -343,6 +392,10 @@ var defaultConfig = Config{
 	LogLevels: LogLevels{
 		UserDaemon: logrus.DebugLevel,
 		RootDaemon: logrus.InfoLevel,
+	},
+	Images: Images{
+		Registry:   "docker.io/datawire",
+		AgentImage: "",
```

Comment:
Shouldn't these defaults be initialized from the current environment variables? I think it's either that or remove them from the env.

Reply:
Ah yeah sorry about that, I hadn't gotten to the point where I removed the environment variables.  Will be in the next commit.